### PR TITLE
Fix #179

### LINKED ISSUE #179: PDB loading is broken
The attached files illustrate the problem. If you provide a to-specification PDB, the numbers in it (Angstrom) are read as they are and thus interpreted as Bohr. Even more worryingly, if you specify `positions{Angstrom}` in the `TITLE` line, you still get Bohr. Only `positions{angstrom}` works if you want to load the file as Angstrom. I specifically remember putting calls to `.lower()` in the units processing to avoid this issue in the input file.

Although we agree that I/O still needs some more work, could someone who worked on the unit conversion have a look at this even before we sort it out for good? I have recently set up several people with i-PI and had to explain carefully how their PDB file had to have an oddly specific first line with quirky capitalization.

[test.zip](https://github.com/cosmo-epfl/i-pi-dev/files/946017/test.zip)

### PR DESCRIPTION
**Pull request by grhawk**
Fixing #179 

Thank you @OndrejMarsalek to pointing out this issues... The problem about "Angstrom" vs "angstrom" was actually due to a regex...

### CODE REVIEW

File: ipi/utils/io/backends/io_pdb.py [hunk 1]
```diff
@@ -116,6 +116,12 @@ def read_pdb(filedesc, **kwargs):
         # skip the comment field
         comment = copy.copy(header)
         header = filedesc.readline()
+    if 'positions{' not in comment:
+        comment = comment.strip()
+        comment += ' positions{angstrom}\n'
```
Comment (OndrejMarsalek) [inline ipi/utils/io/backends/io_pdb.py]: I would still capitalize "Angstrom", at least in the code, even if it does not influence functionality.

Comment (OndrejMarsalek) [inline ipi/utils/io/backends/io_pdb.py]: I would still capitalize "Angstrom", at least in the code, even if it does not influence functionality.

**Comment by ceriottm** [comment]
I would not capitalize angstrom in the code since all units are lowercase.

**Comment by OndrejMarsalek** [comment]
Fair. I guess it just looks weird because normally you only write A and not the whole thing. Anyway, merge?

**Pull request merged**